### ISSUE TITLE
Add Function example establishing db connection

### ISSUE THREAD

**Issue by ccfontes** [ccfontes/faas-bb#13]
Add Function example establishing db connection:
- Pool database connections
- Database connection pool persists between requests
- Database pooling should be established if there isn't one

**Comment by ccfontes** [comment]
https://blog.alexellis.io/serverless-databases-with-openfaas-and-mongo/

**Comment by ccfontes** [comment]
Seems to need HTTP mode to persistent pool of connections between Function requests. Blocked by #7.

Also, not urgent to make this example because folks can use a managed database service that also manages their connection pool.

Example in Node.js: https://github.com/alexellis/mongodb-function

**Comment by ccfontes** [comment]
Auto gen by GPT-4 using Node.js example: https://github.com/ccfontes/faas-bb/blob/add-fn-eg-pool-db-conn/examples/bb-db/handler.clj

**Comment by ccfontes** [comment]
Very low priority. Will re-open if necessary.
Related: https://github.com/ccfontes/faas-bb/pull/60
